frontend/riskofbias/robStudyForm/ScoreForm.js

```diff
@@ -35,10 +36,8 @@ class ScoreInput extends Component {
     }
 }
 ScoreInput.propTypes = {
```

Comment:
@rabstejnek removed scoreId and defaultValue here too, let me know if ok...

Reply:
The purpose of the `defaultValue` here is that when a metric is changed, any wrong score on the score form is automatically defaulted. Without it, we get the original score which will not show the correct score on the select input (since its an invalid option) and will throw an error on form submit if not handled.